STARTERKIT/components/_init.scss

```diff
@@ -52,8 +52,8 @@
 @import 'support-for';
 // Add typey to manage font sizes and margins.
 @import 'typey';
-// Add the Zen Grids responsive layout mixins.
-@import 'zen-grids';
+// Add the Gridlex flexbox library http://gridlex.devlint.fr/.
+@import 'gridlex/src/gridlex-vars';
```

Comment:
It needs at least follow-up to create layouts a-la 
- https://www.drupal.org/project/radix_layouts
- https://www.drupal.org/project/bootstrap_layouts